python_docs_theme/static/pydoctheme.css

```diff
@@ -191,3 +196,254 @@ div.footer a:hover {
 dl > dt span ~ em {
     font-family: monospace, sans-serif;
 }
+
+@media (max-width: 1023px) {
+    /* Body layout */
+    div.body {
+        min-width: 100%;
+        padding: 0;
+        font-size: 0.875rem;
+    }
+    div.bodywrapper {
+        margin: 0;
+    }
+    /* Typography */
+    div.body h1, div.body h2, div.body h3, div.body h4, div.body h5, div.body p {
+        font-family: SourceSansProRegular, Arial, sans-serif;
```

Comment:
Why do you change the font family for a specific width?

Reply:
I removed this change.